imap_processing/lo/l0/compression_tables.py

```diff
@@ -0,0 +1,853 @@
+"""Compression tables for decompompressing Lo L0 data."""
+tof_case_table = {
+    # keys: first 4 bits of a compressed binary
+    # values: case number
+    "0000": 0,
+    "0001": 1,
+    "0010": 2,
+    "0011": 3,
+    "0100": 4,
+    "0101": 5,
+    "0110": 6,
+    "0111": 7,
+    "1000": 8,
+    "1001": 9,
+    "1010": 10,
+    "1011": 11,
+    "1100": 12,
+    "1101": 13,
+}
+
+tof_decoder_table = {
+    # First level of keys are the case numbers
+    0: {
+        1: {
+            "ENERGY": 3,
+            "POS": 0,
+            "TOF0": 10,
+            "TOF1": 0,
+            "TOF2": 9,
+            "TOF3": 6,
+            "CKSM": 3,
+            "TIME": 12,
```

Comment:
Would it make sense to create a namedtuple class for these?
```python
TOFData = namedtuple("TOFData", "ENERGY POS TOF0 TOF1 TOF2 TOF3 CKSM TIME")
{1: TOFData(3, 0, 10, 0, 9, 6, 3, 12)}
```

Reply:
Yeah, I agree, that would be better.